pkg/blockstorage/awsebs/awsebs.go

```diff
@@ -30,8 +30,8 @@ import (
 
 	awsconfig "github.com/kanisterio/kanister/pkg/aws"
 	"github.com/kanisterio/kanister/pkg/blockstorage"
-	ktags "github.com/kanisterio/kanister/pkg/blockstorage/tags"
-	"github.com/kanisterio/kanister/pkg/blockstorage/zone"
+	ktags "github.com/kanisterio/kanister/pkg/blockstorage/utils/tags"
+	"github.com/kanisterio/kanister/pkg/blockstorage/utils/zone"
```

Comment:
Why such deep restructuring?  If you want to move getter, tags and zone into utils flatten it!